entrypoint.sh

```diff
@@ -2,4 +2,4 @@
 
 npm run migrate:docker up || exit 1
 npm run seed || exit 1
-npm run start || exit 1
+node ./dist/src/server.js || exit 1
```

Comment:
are we guaranteed to have the server built in the CI at this point?

Reply:
We do! we actually do a `RUN npm run build` on the Dockerfile